data/releasenotes.json

```
@@ -1,4 +1,25 @@
 [
+	{
+		"version": "v2025-12-19",
+		"date": "2025-12-19",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> Add VACUUM (CHECK_INTEGRITY) to verify database integrity.",
+			"<b>Performance:</b> table scans on partitioned tables are now up to 1000 times faster.",
+			"<b>Performance:</b> improve performance of complex queries on system tables.",
+			"<b>PostgreSQL Compatibility:</b> Support ALTER SEQUENCE operations: RENAME and change OWNER TO.",
```

```suggestion
			"<b>PostgreSQL Compatibility:</b> Support <tt>ALTER SEQUENCE</tt> operations: <tt>RENAME</tt> and <tt>OWNER TO</tt>.",
```
"change" is not part of the syntax, right? I found that a bit confusing